frontend/public/components/instantiate-template.tsx

```diff
@@ -57,7 +58,10 @@ const TemplateInfo: React.FC<TemplateInfoProps> = ({ template }) => {
   const { description } = annotations;
   const displayName = annotations[ANNOTATIONS.displayName] || template.metadata.name;
   const iconClass = getTemplateIcon(template);
-  const imgURL = getImageForIconClass(iconClass);
+  let imgURL = getImageForIconClass(iconClass);
+  if (!iconClass) {
+    imgURL = catalogImg;
+  }
```

Comment:
```suggestion
 const imgURL = iconClass? getImageForIconClass(iconClass) : catalogImg;
```

Reply:
I changed it to `if (!imgURL && !iconClass) {`

IMHO this looks exactly to what it does, it adds a fallback. The problem for me with the ordinary operator is, that it looks that there was only 2 variants. But there are three.

1. Image
2. FA icon class 
3. Fallback

Reply:
> IMHO this looks exactly to what it does, it adds a fallback
@jerolimov yeah my suggestion was not with logic it's more of a nit and looks clean in a line.

Reply:
> I changed it to if (!imgURL && !iconClass) {

do we need both ? isn't `imgURL` derived from `iconClass` ? i.e `imgURL` can only be valid if `iconClass` is. WDYT?